backend/Application/Api/GraphQL/Import/PaydayImportHandler.cs

```diff
@@ -45,8 +45,15 @@ public async Task<BlockImportPaydayStatus> UpdatePaydayStatus(BlockDataPayload p
             }
             else if (status.NextPaydayTime != rewardStatus.NextPaydayTime)
             {
-                var duration = status.NextPaydayTime - status.PaydayStartTime;
-                var result = new FirstBlockAfterPayday(status.NextPaydayTime, Convert.ToInt64(duration.TotalSeconds));
+                var duration = Convert.ToInt64((status.NextPaydayTime - status.PaydayStartTime).TotalSeconds);
+
+                // Upon update to protocol 5 the time was shifted by ~5 secs on Testnet
+                // Duration should be adjusted
+                if(payload.BlockInfo.BlockHash.AsString == "d2d9f88cb953c3314dc1219120d140deaffc44c73ca335f6227b84c09ba7a9d4") {
+                    duration = 24 * 60 * 60;
+                }
+
+                var result = new FirstBlockAfterPayday(status.NextPaydayTime, duration);
```

Comment:
I am not sure this is the right fix. The same will happen on mainnet in the protocol update, and there the block hash will of course be different.

I am not familiar with the data that is here. What are `status.NextPaydayTime` and `status.PaydayStartTime`?

Because what happens is that the first block after the protocol update starts a new payday.

Reply:
Is there any specific way to figure out this change? (Using block summary)

Or can this be considered default behaviour for every protocol change?

Reply:
Allow me sometime to exactly put the table of data here for all the blocks.

Reply:
I don't understand what you mean exactly.

The next payday time for block `47825ccad439af4220d66a5ad55d04f7aa6acd3e230303be1774e33d160778fa` is `2022-11-23T11:05:19.5Z` and for block `d2d9f88cb953c3314dc1219120d140deaffc44c73ca335f6227b84c09ba7a9d4` it is `2022-11-24T11:05:19.5Z` as returned by `GetRewardStatus`. 

Which other blocks are you referring to?

Reply:
Hi @abizjak - the blocks of significance are
Block Height | Next Payday Time
-- | --
1394113 | 24/11/2022 11:05:19
1385893 | 23/11/2022 11:05:19
1385825 | 23/11/2022 11:05:19
1377456 | 22/11/2022 11:00:32
1369077 | 21/11/2022 11:00:32